modules/configuring-firewall.adoc

```diff
@@ -80,22 +71,20 @@ Before you install {product-title}, you must configure your firewall to grant ac
 |`mirror.openshift.com`
 |Required to access mirrored installation content and images
 
-|`storage.googleapis.com`
-|Something
 
-|`cloudfront.net`
-|Required by the Quay CDN
+|`*.cloudfront.net`
+|Required by the Quay CDN to provide edge delivery of Quay.io images
 
-|`https://api.openshift.com/api/upgrades_info/v1/graph`
+|`api.openshift.com/api/upgrades_info/v1/graph`
 |Required to check if updates are available for the cluster
 
 |`art-rhcos-ci.s3.amazonaws.com`
 |Required to download {op-system-first} images
 
-|`https://api.openshift.com/`
+|`api.openshift.com/`
 |Required for your cluster token
 
-|`https://cloud.redhat.com/openshift/`
+|`cloud.redhat.com/openshift/`
 |Required for your cluster token
 
 |`quay-registry.s3.amazonaws.com`
```

Comment:
You may want to move this one up beside redhat registry and *.cloudfront.net. It's the same kind of description "core container images".